charts/develop/jdownloader2/SCALE/questions.yaml

```diff
@@ -0,0 +1,892 @@
+                description: "RollingUpdate: Create new pods and then kill old ones"
+              - value: "OnDelete"
+                description: "(Legacy) OnDelete: ignore .spec.template changes"
+  - variable: env
+    group: "Container Configuration"
+    label: "Image Environment"
+    schema:
+      type: dict
+      attrs:
+        - variable: TZ
+          label: "Timezone"
+          schema:
+            type: string
+            default: "Etc/UTC"
+            $ref:
+              - "definitions/timezone"
+        - variable: PUID
+          label: "PUID"
+          description: "Sets the PUID env var for jdownloader2 containers"
+          schema:
+            type: int
+            default: 568
+        - variable: PGID
+          label: "PGID"
+          description: "Sets the PGID env var for jdownloader2 containers"
+          schema:
+            type: int
+            default: 568
+        - variable: UMASK
+          label: "UMASK"
+          description: "Sets the UMASK env var for jdownloader2 containers"
+          schema:
+            type: string
+            default: "022"
+        - variable: KEEP_APP_RUNNING
+          label: "KEEP_APP_RUNNING"
+          description: "When set to 1, the application will be automatically restarted if it crashes or if a user quits it."
+          schema:
+            type: int
+            default: 0
+        - variable: gui
+          label: "GUI Settings"
+          description: "Always read description before changing a value here. Also refer to README"
+          schema:
+            type: boolean
+            default: false
+            show_subquestions_if: true
+            subquestions:
+              - variable: DISPLAY_WIDTH
+                label: "DISPLAY_WIDTH"
+                description: "Width (in pixels) of the application's window."
+                schema:
+                  type: string
+                  default: "1280"
+                  required: true
+              - variable: DISPLAY_HEIGHT
+                label: "DISPLAY_HEIGHT"
+                description: "Height (in pixels) of the application's window."
+                schema:
+                  type: string
+                  default: "768"
+                  required: true
+              - variable: SECURE_CONNECTION
+                label: "SECURE_CONNECTION"
+                description: "When set to 1, an encrypted connection is used to access the application's GUI (either via a web browser or VNC client). See the Security section for more details."
+                schema:
+                  type: string
+                  default: "0"
+                  required: true
+              - variable: VNC_PASSWORD
+                label: "VNC_PASSWORD"
+                description: "Password needed to connect to the application's GUI. See the VNC Password section for more details."
+                schema:
+                  private: true
+                  type: string
+                  default: ""
```

Comment:
Is this used for J downloader? It seems you copy pasta'd this from handbrake or something.

Reply:
Yeah i have copy pasta'd this from handbrake. But the docker container have all this variables

Reply:
Yea, jd does not have a web portal, thats why it needs the vnc